test/cases/project_hook/event.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": 12867563,
+  "url": "https://example.com/callback",
+  "created_at": "2022-05-31T01:52:14.490Z",
+  "push_events": true,
+  "tag_push_events": false,
+  "merge_requests_events": false,
+  "repository_update_events": false,
+  "enable_ssl_verification": true,
+  "project_id": 36615146,
+  "issues_events": false,
+  "confidential_issues_events": false,
+  "note_events": false,
+  "confidential_note_events": null,
+  "pipeline_events": false,
+  "wiki_page_events": false,
+  "deployment_events": false,
+  "job_events": false,
+  "releases_events": false,
+  "push_events_branch_filter": null
```

Comment:
Just a suggestion, it would be good to have values for both null and non-null fields in the tests.